sw/device/silicon_creator/rom/keys/README.md

```diff
@@ -1,11 +1,55 @@
 # Signing Keys
 
-This directory contains ASN1 DER encoded development keys for signing ROM\_EXT
-images. Until we have a more ergonomic tool for working with keys and signatures,
-below snippets can be used for various operations.
+# Generating keys with opentitantool
 
-Generating a key pair:
+Open Titan Tool is the recommended tool to generate RSA keys to use in
+the `ROM`.
+
+- Inspecting a key (show the public information of a key):
+```console
+$ bazel run //sw/host/opentitantool -- rsa key show $PWD/<basename>.der
+{
+  key_num_bits: 3072,
+  modulus: "0xa22244f7...",
+  public_exponent: "0x010001",
+  n0_inv: "0x2af3...",
+  rr: "0x82ca57ba..."
+}
 ```
+
+- Generating a key pair in the current directory:
+```console
+$ bazel run //sw/host/opentitantool -- rsa key generate <outdir> <basename>
```

Comment:
```suggestion
```sh
bazel run //sw/host/opentitantool -- rsa key generate $outdir $basename
```

* I think removing `$` would allow for easier copy and pasting for users.
* Also if `basename` is a variable, people can set the variable and are then able to copy and paste all the commands. `sh` highlights variables nicely too.

This is just an opinion though, so there is no need to block this PR. I'll open a PR adding to the style guide to see if there is any consensus on this stuff.